pkg/virt-controller/services/virtiofs.go

```diff
@@ -116,18 +99,51 @@ func generateContainerFromVolume(volume *v1.Volume, image string, resources k8sv
 		},
 	}
 
+	volumeMountPath := virtiofs.FSMountPoint(volume)
 	if !isAutoMount(volume) {
 		volumeMounts = append(volumeMounts, k8sv1.VolumeMount{
 			Name:      volume.Name,
-			MountPath: virtioFSMountPoint(volume),
+			MountPath: volumeMountPath,
+		})
+	}
+
+	var (
+		cmd  []string
+		args []string
+	)
+
+	switch {
+	case utils.IsStorageVolume(volume):
+		cmd = []string{"/usr/bin/virtiofs-placeholder"}
+		args = []string{"--socket", virtiofs.PlaceholderSocketPath(volume.Name)}
+
+		volumeMounts = append(volumeMounts, k8sv1.VolumeMount{
+			Name:      virtiofs.PlaceholderSocketVolumeName,
+			MountPath: virtiofs.PlaceholderSocketVolumeMountPoint,
 		})
+	case utils.IsConfigVolume(volume):
+		cmd = []string{"/usr/libexec/virtiofsd"}
+
+		socketPathArg := fmt.Sprintf("--socket-path=%s", virtiofs.VirtioFSSocketPath(volume.Name))
+		sourceArg := fmt.Sprintf("--shared-dir=%s", volumeMountPath)
```

Comment:
nit: you could concat these strings with fmt.Sprintf

Reply:
Yes, but I think is ore clear this way, since go doesn't string interpolation I'll be forced to write something like:
```
fmt.Sprintf("--socket-path=", virtiofs.VirtioFSSocketPath(volume.Name, " --shared-dir=", volumeMountPath)
```
that is more error-prone, same with positional arguments and `%s`

Reply:
sorry, I meant without.

e.g. `"--shared-dir=" + volumeMountPath`